### ISSUE TITLE
Could you please share the code with the loss of Understanding Deformable Alignment in Video Super-Resolution in EDVR?

### ISSUE THREAD

**Issue by wenchen4321** [xinntao/EDVR#175]
I have read the paper named Understanding Deformable Alignment in Video Super-Resolution.  I am very interested in the Offset-ﬁdelity Loss.  Could you please share the code with the Offset-ﬁdelity Loss in EDVR?

**Comment by xinntao** [comment]
Hi @wenchen4321  
Thanks for your interests. 
We are sorting out our codes. We will release when it is ready.

**Comment by csbhr** [comment]
I am also very interested in the offset-fidelity loss presented in "Understanding Deformable Alignment in Video Super-Resolution". It is awesome to use optical flow to stabilize the training process.

Could you please release the codes of this offset-fidelity loss, or share more implementation details, such as:
How to determine the hyperparameters (i.e. "lamda" and "t" in Eq4 and Eq5 in the paper) ?
Whether the offsets of each level will be constrained? 
How to distinguish the corresponding direction of optical flow when constraining the two directions of offsets?

Look forward to your reply, thank you!
